src/git.rs

```diff
@@ -42,4 +89,22 @@ mod tests {
         let post_fetch_branches = git_remote_branches(&test_git.local);
         assert_eq!(post_fetch_branches.count(), 3); // HEAD, main and other
     }
+
+    #[test]
+    fn can_list_git_branches() {
+        let test_git = initialise_git_repo(None);
+        git_fetch(&test_git.dir.path().join("local")).unwrap();
+        let branches = git_list_branches(&test_git.dir.path().join("local")).unwrap();
+        let now_in_seconds = SystemTime::now()
```

Comment:
This is going to be brittle, if the second rolls over between the call to `initialise_git_repo` and this line. I'm not sure it really is essential to test that value, but if you want to you'd have to do something like take the time before and after and check that it falls in the range.

Reply:
i dont mind just omitting the test for time value, feels fiddly with not much gain